src/test/java/com/github/sdonkov/PrefixTreeTest.java

```diff
@@ -25,6 +27,18 @@ static void setUp() throws IOException, URISyntaxException {
         words.forEach(word -> prefixTree.add(word));
     }
 
+    @Test
```

Comment:
This is a good start, but what we want from the unit-tests is full coverage of the public API of the class (which I think we have right now on master).
It'd be great if you can measure test coverage in your IDE and post the result before and after the change.